### ISSUE TITLE
cannot blit_into image 

### ISSUE THREAD

**Issue by emrum** [pyglet/pyglet#787]
**Describe the bug**
for some reason , image . blit_into( source , ...)  calls an abstract method and throws exception. 

**System Information:**
this is for pyglet version 2.0.5 

**How To Reproduce**
here is the code:

import pyglet
from pyglet.gl import *

window = pyglet.window.Window()

tex_big = pyglet.image.load('texture_RGBA.png').get_texture()

tex_small = pyglet.image.load('white_pixel.png').get_texture()

@window.event
def on_draw():

    glClearColor(255,255,255,255)
    # Clear the window
    window.clear()

    tex_big .blit_into (source=tex_small, x=0, y=0, z=0)

    tex_big .blit (0,0,0)
    
#
pyglet.app.run()

**Comment by caffeinepills** [comment]
The source has to be `ImageData`, not a `Texture`.

**Comment by emrum** [comment]
> The source has to be `ImageData`, not a `Texture`.

ah , this.  then why it is not in the documentation ?  I had tried in vain, with image as well.

**Comment by emrum** [comment]
true,  it works with get_imge_data() ,  seems this time, the bug is more in the documentation and error-msg, wich do not tell anything about it.
